### DISCUSSION TITLE
Notification panel is not closing after action.

### DISCUSSION THREAD

**Discussion by philipbaginski** [laravel/nova-issues#5055] [Ideas & Feature Requests]
- Laravel Version: 9.39.0
- Nova Version: 4.18.2 (Silver Surfer)
- PHP Version: 8.0.3
- Database Driver & Version: 5.7.24
- Operating System and Version: Windows NT (10.0 - build 18363 (Windows 10))
- Browser type and version: Chrome 107.0.5304.88
- 
Detailed steps to reproduce the issue on a fresh Nova installation:

So, the issue is about: The notification is fired. The bell is getting the red dot. Clicking on the bell opens notifications panel. We see notification/s. In the notification are a message and an action link. And... clicking the action button takes us to the resource the notification was created because, but notification panel is still open. We have to click outside of the panel to close the panel. It should not be like that :-) The panel should be closed automatically in the same moment we are clicking action button.

Added: And also, if button action is clicked, it is probably clear sign, that this notification is read, so, automatically this notification should be marked as read.

My sample code:

    /**
     * @param  Workout $user
     */
    private function getNovaNotification($workout, $message, $actionMessage, $type, $url): void
    {
        foreach(User::all() as $user)
        $user->notify(
        NovaNotification::make()
            ->message($message.' '.$workout->title)
            ->action($actionMessage, URL::make("/resources/workouts/{$url}"))
            ->icon('clock')
            ->type($type)
        );
    }

**Comment by divdax** [comment]
I also expect the notification panel to be closed when I click an action. Did you find a solution?

**Comment by henryavila** [comment]
Same problem here. The user navigate to the page but has to manually close the panel

**Comment by tmartty** [comment]
This has been fixed in one of the latest Nova versions and the notifications panel now automatically closes. But the clicked notification does still show as "not read".
